webapp/src/workspace.ts

```diff
@@ -121,23 +121,21 @@ export function restoreFromBackupAsync(h: Header) {
     return getTextAsync(refId)
         .then(files => {
             delete h.backupRef;
-            return saveAsync(h, files);
+            return forceSaveAsync(h, files);
         })
         .then(() => {
             const backup = getHeader(refId);
             backup.isDeleted = true;
-            return saveAsync(backup);
+            return forceSaveAsync(backup);
         })
         .catch(() => {
             delete h.backupRef;
-            return saveAsync(h);
+            return forceSaveAsync(h);
```

Comment:
Do all these force saves have any implications on performance or increase the chance of race conditions from double saves or something similar?

Reply:
force save means that we override the lock (it has no perf implication) in this case.